src/xdist/remote.py

```diff
@@ -24,6 +24,24 @@ def setproctitle(title):
         pass
 
 
+class Producer:
```

Comment:
We should note that this is a shim of the legacy py api that we couldn't remove at once

Reply:
Agree, will leave a suggestion below.